en/08_Changelogs/6.0.0.md

```diff
@@ -184,6 +184,13 @@ If that happens, you may be able to resolve it with one of the following solutio
 - Write a `BuildTask` that updates any invalid values in your database.
 - While it's generally not recommended, you have the option of disabling validation via the [`DataObject.validation_enabled`](api:SilverStripe\ORM\DataObject->validation_enabled) configuration property.
 
+In addition, the following `DBField` subclasses have also had the follow changes made to them:
+
+- [`DBBoolean`](api:SilverStripe\ORM\FieldType\DBBoolean) will no longer convert non-boolean like values to a `bool` when calling `setValue()`. For instance `"Amazing"` will no longer be converted to `true`, instead its value will remain as `"Amazing"` and it will throw a validation exception if attempted to be written to the database. There are some boolean-like values, such as `"t"`, that will continue to be converted to their boolean equivalents for backwards compatibility. See [validation added to `DBField`s](#dbfield-validation) for specifics.
```

Comment:
```suggestion
- [`DBBoolean`](api:SilverStripe\ORM\FieldType\DBBoolean) will no longer convert non-boolean like values to a `bool` when calling `setValue()`. For instance `"Amazing"` will no longer be converted to `true`, instead its value will remain as `"Amazing"` and it will throw a validation exception if attempted to be written to the database. There are some boolean-like values, such as `"t"`, that will continue to be converted to their boolean equivalents for backwards compatibility.
```
Now that it is in this section we don't need this link

Reply:
Updated